### ISSUE TITLE
Doesn't work now

### ISSUE THREAD

**Issue by deadcoder0904** [zcaceres/medium-megaclap#1]
I think they changed the [API URL](https://github.com/zcaceres/medium-megaclap/blob/master/src/index.js#L21) as its a 404.

**Comment by zcaceres** [comment]
Yep you're right. Looks like Medium moved to GraphQL – PRs welcome ;-)
